.github/actions/verify-release/action.yml

```diff
@@ -0,0 +1,59 @@
+name: Run Cypress integration tests on a remote environment
+description: Run integration tests on a remote environment
+inputs:
+  deploy-env:
+    description: The environment being deployed (e.g. "prod" or "test")
+    required: true
+runs:
+  using: composite
+  steps:
+    - name: Use Node.js ${{env.NODE_VERSION}}
+      uses: actions/setup-node@v2.1.5
+      with:
+        node-version: 14
+    - name: Cache yarn
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/yarn
+        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-yarn-
+    - name: Cache Cypress binary
+      uses: actions/cache@v2
+      with:
+        path: ~/.cache/Cypress
+        key: cypress-${{ runner.os }}-cypress-${{ hashFiles('**/package.json') }}
+        restore-keys: |
+          cypress-${{ runner.os }}-cypress-
+    - name: Makefile deploy verification for backend in ${{env.DEPLOY_ENV}} environment
+      shell: bash
+      working-directory: ./ops
+      env:
+          CURL_TIMEOUT: 5'
+      run: |
+        echo "::group::Run Makefile verification"
+        make check-${{env.DEPLOY_ENV}}-commit check-${{env.DEPLOY_ENV}}-readiness
+        echo "::endgroup::"
+      - name: install dependencies and verify Cypress
+      shell: bash
+      working-directory: ./frontend
+      env:
+        # make sure every Cypress install prints minimal information
+        CI: 1
+      # print Cypress and OS info
+      run: |
+        yarn install --prefer-offline
```

Comment:
I pulled this pattern from our e2e workflow. I propose creating a separate ticket to revisit the strategy around this.

This ticket already feels like it's crept in scope quite a bit, and it would be great to get it closed out.

Reply:
Resolving per our discussion (create future ticket to handle this)